config/vpc/config.go

```diff
@@ -1,14 +1,15 @@
-package v1alpha1
+package vpc
 
 import (
 	"github.com/crossplane-contrib/terrajet/pkg/config"
 )
 
-func vpcExternalNameConfigure(_ map[string]interface{}, _ string) {}
+func VpcExternalNameConfigure(_ map[string]interface{}, _ string) {}
 
 var vpcCustomConfig = config.Resource{
+	Kind: "Vpc",
```

Comment:
```suggestion
	Kind: "VPC",
```

Reply:
I am not sure on this. We still have fields including `Vpc` since we don't want to use acronyms in fields. For example, if a resource has a ref to vpc it will have a field like `Vpc`, `VpcRef`, `VpcSelector`.
So, I feel like being consistent is more important.